### ISSUE TITLE
My skin is not being uploaded on my Ely.by account

### ISSUE THREAD

**Issue by GTHB-plyr7510** [MultiMC/Launcher#5208]
### Description of bug

**What I expected it to happen:**
The skin will be uploaded like normal.
**What I got instead**
MultiMC shows a "Failed to upload skin!" pop-up, and my skin was not changed.
<img width="446" alt="image" src="https://github.com/MultiMC/Launcher/assets/138413585/2b1d2d86-ddc2-43f3-8022-f9410b04c8e9">
This was the skin I tried to upload:
![SPACE](https://github.com/MultiMC/Launcher/assets/138413585/06977526-dc96-465f-b653-d0c536141ce2)

### Steps to reproduce

1. Use an Ely.by account
2. Go to settings > Accounts > Your Ely.by account
3. Click "Upload skin"
4. Click the button with the 3 dots and select your PNG
5. Click OK

### Suspected cause

I do not know what the cause of this bug is. I checked the console, nothing. Searched the issue tracker, nothing. But still I could not find a solution to the problem.

Oh, and I could try to upload logs, but I don't know how. Can anybody help me with that too?

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by GTHB-plyr7510** [comment]
Edit: This also happened on my local account too.

**Comment by phit** [comment]
you aren't using multimc, take this elsewhere

**Comment by GTHB-plyr7510** [comment]
> you aren't using multimc

You're right. I used the Github download link instead of the one on MultiMC's website. So I downloaded the REAL MultiMC, connected my Xbox account, tried to upload a skin and... no luck. The same issue still happened. guess I have to create a new issue or wait until you respond.
<img width="835" alt="image" src="https://github.com/MultiMC/Launcher/assets/138413585/a89fdb4f-eb79-4661-aa4d-792a92db7d13">
